src/controller/CHIPDeviceController.cpp

```diff
@@ -2372,6 +2374,32 @@ CHIP_ERROR DeviceCommissioner::ParseICDInfo(ReadCommissioningInfo & info)
         }
     }
 
+    if (!isICD)
+    {
+        return CHIP_NO_ERROR;
+    }
+
+    err = mAttributeCache->Get<IcdManagement::Attributes::IdleModeDuration::TypeInfo>(kRootEndpointId, info.icd.idleModeDuration);
+    if (err != CHIP_NO_ERROR)
+    {
+        ChipLogError(Controller, "IcdManagement.IdleModeDuration expected, but failed to read: %" CHIP_ERROR_FORMAT, err.Format());
+        return err;
+    }
+    err =
+        mAttributeCache->Get<IcdManagement::Attributes::ActiveModeDuration::TypeInfo>(kRootEndpointId, info.icd.activeModeDuration);
+    if (err != CHIP_NO_ERROR)
+    {
+        ChipLogError(Controller, "IcdManagement.ActiveModeDuration expected, but failed to read.");
```

Comment:
To repeat from last time: Please include err.Format() in all these error logs.

Emphasis on _all_.  It got added to the one specific log I commented on, not _all_.

Reply:
fixed across in my pr, thanks